Example SCIM provider

ccoors opened the pull request:
~This is the current state of my example SCIM provider WSGI application.~

Moved implementation to [scim2-server](https://github.com/yaal-coop/scim2-server). Added as submodule here.

azmeuk commented:
Hi @ccoors
Thank you for your contribution! I will try to gather enough time to review your PR in the coming days/weeks.

ccoors commented:
I have managed to implement PATCH :) Now working on Sorting and will probably also do Bulk.

azmeuk commented:
Hi. I still could not find much time for reviewing your work, but I don't lose hope!

azmeuk commented:
I was thinking that maybe we could add your example server implementation in another repo.
To keep the codebase synchronized with scim2-models, we could then add it as a subrepository here.
If it is good for you, I will create a `scim2-example-server` repository and give you commit permissions. What do you think?

ccoors commented:
Yes, that sounds fine. Git submodules are a bit annoying to work with, but in this instance I guess it's okay.

azmeuk commented:
Ok. I invited you on https://github.com/yaal-coop/scim2-server

ccoors commented:
@azmeuk I added the server in scim2-server and modified this PR, so that it uses it as a submodule. I'm not sure this will provide an 'easy' workflow for future changes in scim2-models, as you may have to synchronize changes across two repos. Maybe try it out for now?

azmeuk commented:
Closing in favor of #71
